decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -258,16 +320,35 @@ def gem_modifier
         @gem_modifier ||= if options[:path]
                             %(path: "#{options[:path]}")
                           elsif branch.present?
-                            %(git: "https://github.com/decidim/decidim.git", branch: "#{branch}")
+                            %(git: "#{repository}", branch: "#{branch}")
                           else
-                            %("#{Decidim::Generators.version}")
+                            %(git: "#{repository}", branch: "#{branch}" ) # TODO: remove before merge
+                            # %("#{Decidim::Generators.version}")
                           end
       end
 
       def branch
         return if options[:path]
 
-        @branch ||= options[:edge] ? "develop" : options[:branch].presence
+        "feat/env-vars-for-services" # TODO: remove before merge
+        # @branch ||= options[:edge] ? "develop" : options[:branch].presence
```

Comment:
```suggestion
        @branch ||= options[:edge] ? "develop" : options[:branch].presence
```
TODO: Before merging